README.md

```diff
@@ -1,107 +1,157 @@
 
 ```yaml
-
 jobs:
-  build_test_scan_publish:
-    runs-on: (Use our custom runner)
+  build_test_scan:
+    runs-on: eks
     name: CI Pipeline
     steps:
     - uses: actions/checkout@v2
       with:
-        fetch-depth: 0         
+        fetch-depth: 0
+        
+    - name: Setup
+      uses: variant-inc/lazy-action-setup@v1
+                  
     - name: Lazy action steps
       id: lazy-action
-      uses: variant-inc/lazy-action@master
+      uses: variant-inc/lazy-action-dotnet@v1
+      env:
+        NUGET_TOKEN: ${{ secrets.PKG_READ }}
+        AWS_DEFAULT_REGION: us-east-2
+        GITHUB_USER: variant-inc
       with:
-        src_file_dir: '.'
+        src_file_dir_path: '.'
         dockerfile_dir_path: '.'
-        github_token: ${{ secrets.PKG_READ }}
-        github_owner_token: ${{ secrets.GITHUB_TOKEN }}
-        sonar_token: ${{ secrets.SONAR_TOKEN }}
-        aws_access_key_id: ${{ secrets.DEV_AWS_ACCESS_KEY_ID }}
-        aws_secret_access_key: ${{ secrets.DEV_AWS_SECRET_ACCESS_KEY }}
-        docker_repo_name: 'demo-app'
-        docker_image_name: 'demo-repo'
-        nuget_push: 'true'
-        docker_push: 'true'
-        sonar_scan_enabled: 'false'
+        ecr_repository: naveen-demo-app/demo-repo
+        nuget_push_enabled: 'true'
+        sonar_scan_in_docker: 'false'
+        nuget_src_project: "src/Variant.ScheduleAdherence.Client/Variant.ScheduleAdherence.Client.csproj"
+        nuget_package_name: 'demo-app'
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Lazy Action Octopus
+      uses: variant-inc/lazy-action-octopus@v1
+      with:
+        default_branch: ${{ env.MASTER_BRANCH }}
+        deploy_scripts_path: deploy
+        project_name: ${{ env.PROJECT_NAME }}
+        version: ${{ steps.lazy-setup.outputs.image-version }}
+        space_name: ${{ env.OCTOPUS_SPACE_NAME }}
 
 ```
 
 ## parameters
 
 ### Input Parameters
 
+| Parameter                    | Default       | Description                                           | Required |
+| ---------------------------- | ------------- | ----------------------------------------------------- | -------- |
+| `src_file_dir_path`          | `.`           | Directory path to the solution file                   | true     |
+| `dockerfile_dir_path`        | `.`           | Directory path to the dockerfile                      | true     |
+| `ecr_repository`             |               | ECR Repository name                                   | true     |
+| `sonar_scan_in_docker`       | "false"       | Is sonar scan running as part of Dockerfile           | false    |
+| `sonar_scan_in_docker_target`|"sonarscan-env"| sonar scan in docker target.                          | false    |
+| `nuget_push_enabled`         | "false"       | Enabled Nuget Push to Package Registry.               | false    |
+| `nuget_package_name`         |               | Creates the nuget package with this name.Used only when nuget_push_enabled is true.| false |
```

Comment:
Please make sure the grammar is correct. There should be space after a sentence is ended